src/student.rb

```diff
@@ -4,9 +4,15 @@ class Student < Person
   def initialize(*args, classroom)
     super(*args)
     @classroom = classroom
+    update_classroom(@classroom)
   end
+  attr_accessor :classroom
```

Comment:
- [ ] **Optional:** ⬆️ Always at the top of the class in future projects, please.